src/main/java/io/supertokens/authRecipe/AuthRecipe.java

```diff
@@ -54,4 +54,11 @@ public static UserPaginationContainer getUsers(Main main, Integer limit, String
         System.arraycopy(users, 0, resultUsers, 0, maxLoop);
         return new UserPaginationContainer(resultUsers, nextPaginationToken);
     }
+
+    public static void deleteUser(Main main, String userId) throws StorageQueryException {
+        StorageLayer.getSessionStorage(main).deleteSessionsOfUser(userId);
+        StorageLayer.getEmailVerificationStorage(main).deleteEmailVerificationUserInfo(userId);
+        StorageLayer.getEmailPasswordStorage(main).deleteEmailPasswordUser(userId);
+        StorageLayer.getThirdPartyStorage(main).deleteThirdPartyUser(userId);
```

Comment:
Might wanna add a comment explaining here that the order of deletion is important and why. So that in the future, this doesn't get changed.